src/integration-test/system-integration-tests.ts

```diff
@@ -280,46 +272,55 @@ describe('system-integration', () => {
           });
         });
 
+        // TODO: for unknown reasons these fail pretty much randomly, depending on the machine and luck.
+        // If the team-scoped events are skipped, then these suddenly pass.
+        // Also when using the LocalEventBus these pass, including persistence.
```

Comment:
Realtime events won't, and shouldn't, actually work at all when using the LocalEventBus. Fixed.